.github/workflows/go.yml

```
@@ -15,6 +15,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v1
+        with:
+          go-version: "1.17.6"
```

Do we depend on a specific version of Go for this feature to work? Maybe it's worth upgrading the `actions/setup-go` to v4 and using `go-version: 'stable'`?
```suggestion
      - uses: actions/setup-go@v4
        with:
          go-version: "stable"
```